common/djangoapps/third_party_auth/strategy.py

```diff
@@ -77,3 +77,19 @@ def create_user(self, *args, **kwargs):
 
         # when autoprovisioning we need to skip email activation, hence skip_email is True
         return create_account_with_params(self.request, user_fields, skip_email=True)
+
+    def request_host(self):
```

Comment:
Nit: I think we should add a comment here saying that this is a temporary measure until it is fixed upstream in python-social-auth, and that this is not required if using gunicorn.